src/compass/utils/h5_helpers.py

```diff
@@ -446,3 +427,80 @@ def poly1d_to_h5(group, poly1d_name, poly1d):
+    rg_lut, az_lut = compute_geocoding_correction_luts(burst,
+                                                       rg_step=cfg.lut_params.range_spacing,
+                                                       az_step=cfg.lut_params.azimuth_spacing)
+
+    desc = ' correction as a function of slant range and azimuth time'
+    add_dataset_and_attrs(correction_group, 'azimuth_LUT', az_lut.data,
+                          {'description': f'azimuth {desc}',
+                           'units': 'seconds'})
+    add_dataset_and_attrs(correction_group, 'slant_range_LUT', rg_lut.data,
+                          {'description': f'slant_range {desc}',
+                           'units': 'seconds'})
+
+    # az/rg_lut share the same grid
+    slant_range = np.linspace(az_lut.x_start,
+                              az_lut.x_start + az_lut.width * az_lut.x_spacing,
+                              az_lut.width, dtype=np.float64)
+    add_dataset_and_attrs(correction_group, 'slant_range', slant_range,
+                          {'description': 'slant range of LUT data',
+                           'units': 'meters'})
+
+    azimuth = np.linspace(az_lut.y_start,
+                          az_lut.y_start + az_lut.width * az_lut.x_spacing,
+                          az_lut.width, dtype=np.float64)
+    add_dataset_and_attrs(correction_group, 'azimuth_time', azimuth,
+                          {'description': 'azimuth time of LUT data',
+                           'units': 'seconds'})
+
+    # EAP metadata depending on IPF version
+    check_eap = is_eap_correction_necessary(burst.ipf_version)
+    if check_eap.phase_correction:
+        eap_group = correction_group.require_group('elevation_antenna_pattern_correction')
+        eap = burst.burst_eap
+        add_dataset_and_attrs(eap_group, 'sampling_frequency',
+                              eap.freq_sampling,
+                              {'description': 'range sampling frequency',
+                               'units': 'Hz'})
+        add_dataset_and_attrs(eap_group, 'eta_start',
+                              eap.eta_start.strftime(TIME_STR_FMT),
+                              {'description': '?',
+                               'format': 'YYYY-MM-DD HH:MM:SS.6f'})
+        add_dataset_and_attrs(eap_group, 'tau_0', eap.tau_0,
+                              {'description': 'slant range time',
+                               'units': 'seconds'})
+        add_dataset_and_attrs(eap_group, 'tau_sub', eap.tau_sub,
+                              {'description': 'slant range time',
+                               'units': 'seconds'})
+        add_dataset_and_attrs(eap_group, 'theta_sub', eap.theta_sub,
+                              {'description': 'elevation angle',
+                               'units': 'radians'})
+        add_dataset_and_attrs(eap_group, 'azimuth_time',
+                              eap.azimuth_time.strftime(TIME_STR_FMT),
+                              {'description': '?',
```

Comment:
realized that the member `BurstEAP.azimuth_time` is redundant (i.e. same value as `BurstEAP.eta_start`.
I'll remove this member in the follow-up PR to `s1-reader`.